checks.d/linux_proc_extras.py

```diff
@@ -0,0 +1,70 @@
+# project
+from checks import AgentCheck
+from utils.subprocess_output import get_subprocess_output
+from collections import Counter
+
+PROCESS_STATES = {
+    'D': 'uninterruptible',
+    'R': 'runnable',
+    'S': 'sleeping',
+    'T': 'stopped',
+    'W': 'paging',
+    'X': 'dead',
+    'Z': 'zombie',
+}
+
+PROCESS_PRIOS = {
+    '<': 'high',
+    'N': 'low',
+    'L': 'locked'
+}
+
+class MoreUnixCheck(AgentCheck):
+    def check(self, instance):
+        tags = instance.get('tags', [])
+
+        state_counts = Counter()
```

Comment:
I know i'm the one who told you to use a Counter but i forgot that Counter were not available on python 2.6. Can you use a `defaultdict(int)` instead ?